src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/generators/CoalGenerator.java

```diff
@@ -35,8 +42,44 @@ protected void registerDefaultFuelTypes() {
         for (Material mat : Tag.PLANKS.getValues()) {
             registerFuel(new MachineFuel(1, new ItemStack(mat)));
         }
+
+        // Wooden Slabs
+        for (Material mat : Tag.WOODEN_SLABS.getValues()) {
+            registerFuel(new MachineFuel(1, new ItemStack(mat)));
+        }
+
+        // Wooden Buttons
+        for (Material mat : Tag.WOODEN_BUTTONS.getValues()) {
+            registerFuel(new MachineFuel(1, new ItemStack(mat)));
+        }
+
+        // Wooden Fences
+        for (Material mat : Tag.WOODEN_FENCES.getValues()) {
+            registerFuel(new MachineFuel(1, new ItemStack(mat)));
+        }
+
+        // wooden Trapdoors
+        for (Material mat : Tag.WOODEN_TRAPDOORS.getValues()) {
+            registerFuel(new MachineFuel(1, new ItemStack(mat)));
+        }
+
+        // Wooden Pressure Plates
+        for (Material mat : Tag.WOODEN_PRESSURE_PLATES.getValues()) {
+            registerFuel(new MachineFuel(1, new ItemStack(mat)));
+        }
+
+        // Wooden Doors
+        for (Material mat : Tag.WOODEN_DOORS.getValues()) {
+            registerFuel(new MachineFuel(1, new ItemStack(mat)));
+        }
+
+        // Signs
+        for (Material mat : Tag.SIGNS.getValues()) {
+            registerFuel(new MachineFuel(1, new ItemStack(mat)));
```

Comment:
You use 6 planks to craft 3 signs, so it should be 2 here.

Reply:
i got this info straight from the wiki https://minecraft.gamepedia.com/Smelting im fine changing it if you actually want to have smelting time differ from mc official time

Reply:
I'd rather have them make sense than be accurate :LUL: